bin/plugin/commands/performance.js

```diff
@@ -95,23 +113,25 @@ async function runPerformanceTests( branches, options ) {
 		branches = [ 'trunk' ];
 	}
 
+	log( fTitle( '\n💃 Performance Tests 🕺' ) );
 	log(
-		formats.title( '\n💃 Performance Tests 🕺\n' ),
-		'\nWelcome! This tool runs the performance tests on multiple branches and displays a comparison table.\n' +
-			'In order to run the tests, the tool is going to load a WordPress environment on ports 8888 and 8889.\n' +
-			'Make sure these ports are not used before continuing.\n'
+		'\nWelcome! This tool runs the performance tests on multiple branches and displays a comparison table.'
 	);
 
 	if ( ! runningInCI ) {
+		log(
+			fWarning(
+				[
+					'\nIn order to run the tests, the tool is going to load a WordPress environment on ports 8888 and 8889.',
+					'Make sure these ports are not used before continuing.\n',
+				].join( '\n' )
+			)
+		);
```

Comment:
personally I don't find it clearer what we're doing here, either in replacing `formats.title` with `fTitle` (because now I have to double-check what `fTitle` is, only to realize it's nothing more than `formats.title`) or with `.join()`ing lines that were previously concatenated via language syntax.

the join not only isn't eradicating the embedded newlines, but it's also not doing anything but adding _more_ lines and runtime code to what was there before.

is the reason for this change that the auto-formatter is indenting the previously successive lines?

Reply:
Yeah, although it made more sense when used for more than just 2 lines. Will revert to just adding the lines. 👍 

The `f[format]` aliases were to prevent further formatting backflips. I can see the argument for clarity here, though, so I'll try to maybe ignore prettier instead.

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/54309/commits/52d04961ca96f2dd94dec02080a4cfc227a3e0a1